### DISCUSSION TITLE
Place to store our model?

### DISCUSSION THREAD

**Discussion by cahya-wirawan** [facebookresearch/ParlAI#4365] [Q&A]
Hi, is there a place to store our fine tuned model for public similar like model hub in Huggingface?

**Comment by klshuster** [comment]
Yes, you can add your model to the [model zoo](https://github.com/facebookresearch/ParlAI/tree/main/parlai/zoo); you can add your model to the `model_list.py`, and take a look at various models within that directory for how to add build/load files. Then, your model can be instantiated with `zoo:model_dir/model_name/model`

**Reply by cahya-wirawan** [reply]
does it means, if I have a new model I can send a PR to add it in model/zoo? how about the model binary?  I see parlai download it from http://parl.ai/downloads/_models/, how can I store it there?

**Reply by stephenroller** [reply]
We don't usually host third party models on our servers. Most commonly, people upload to a Google drive link. Our Download item has support for downloading from gdrive links.
